src/components/Tables/ComparisonTable.vue

```diff
@@ -117,12 +120,16 @@
             'header-row-4': Unittype.length > 0 && QuotationName.length === 2, 
             'header-row-3': Unittype.length > 0 && QuotationName.length > 2
           }">
-            <th colspan="6"   class="sticky-col"></th>
+            <th class="sticky-col"></th>
```

Comment:
sticky column already have?

Reply:
There already is , i remove out the colspan='6'